### ISSUE TITLE
Specify NodeSelector for the addon deployment [Feature]

### ISSUE THREAD

**Issue by VinayGoudCheguri** [kubevela/kubevela#5362]
Can we provide node selector while enabling the add-ons?
In the k8s cluster I use, there are validation webhooks that do not permit creating deployments without specifying node selector or node affinity.
It will be convenient if we can specify node selector in the below command which is used for enabling addons.
`vela addon enable velaux`

I'm using the Kubernetes job to execute the above command taking vela-cli as the base image.
Please suggest if there are any alternatives like specifying the default node selector for all the add-ons.

**Comment by MdSahil-oss** [comment]
@Somefive I would like to work on it. Can you assign this issue to me.

**Comment by TheRealSibasishBehera** [comment]
Hi @Somefive I am new to this project , i would like to look into this issue
@MdSahil-oss if its alright with with you we can work on it together  :slightly_smiling_face:

**Comment by MdSahil-oss** [comment]
@TheRealSibasishBehera Thanks but I'm almost done with this issue and going to drop a PR for this issue :)

**Comment by bhavyastar** [comment]
Hey @Somefive , I want to work on this issue. If you are not done yet, Can I work on this issue? @MdSahil-oss

**Comment by MdSahil-oss** [comment]
Yeah! sure @bhavyastar you can work on it. I'm busy on other enhancements.

**Comment by wonderflow** [comment]
@bhavyastar please go ahead, thanks

**Comment by prnvtripathi** [comment]
Hi @wonderflow ,
I’m interested in working on this issue.
